app.js

```diff
@@ -0,0 +1,55 @@
+const dotenv = require("dotenv");
+const express = require("express");
+const expressLayouts = require("express-ejs-layouts");
+const flash = require("connect-flash");
+const mongoose = require("mongoose");
+const passport = require("passport");
+require("./middleware/passport")(passport);
```

Comment:
You can require this one after all the assigning ones.

Reply:
updated